tests/astimezone_test.py

```diff
@@ -0,0 +1,247 @@
+    # UTC to JST (UTC+9)
+    jst_result = utc_dt.astimezone(ZoneInfo('Asia/Tokyo'))
+    assert jst_result.hour == 21  # 12:00 UTC + 9 hours = 21:00 JST
+
+
+def test_astimezone_without_arguments() -> None:
+    """Test that astimezone without arguments defaults to local timezone."""
+    # Test with datetime_utc
+    utc_dt = datetime_utc(2021, 1, 1, 10, 0, 0, tzinfo=ZoneInfo('UTC'))
+    result = utc_dt.astimezone()  # No arguments
+
+    assert isinstance(result, datetime_tz)
+    # Should convert to local timezone
+    assert result.tzinfo is not None
+    assert result.tzinfo != ZoneInfo('UTC')  # Should not be UTC anymore
+
+    # Test with datetime_cet
+    cet_dt = datetime_cet(2021, 1, 1, 10, 0, 0, tzinfo=ZoneInfo('CET'))
+    result_cet = cet_dt.astimezone()  # No arguments
+
+    assert isinstance(result_cet, datetime_tz)
+    # Should convert to local timezone
+    assert result_cet.tzinfo is not None
+    assert result_cet.tzinfo != ZoneInfo('CET')  # Should not be CET anymore
+
+    # Test with datetime_tz
+    tz_dt = datetime_tz(2021, 1, 1, 10, 0, 0, tzinfo=ZoneInfo('UTC'))
+    result_tz = tz_dt.astimezone()  # No arguments
+
+    assert isinstance(result_tz, datetime_tz)
+    # Should convert to local timezone
+    assert result_tz.tzinfo is not None
+    assert result_tz.tzinfo != ZoneInfo('UTC')  # Should not be UTC anymore
+
+
+def test_astimezone_with_explicit_none() -> None:
+    """Test that astimezone with explicit None argument defaults to local timezone."""
+    # Test with datetime_utc
+    utc_dt = datetime_utc(2021, 1, 1, 10, 0, 0, tzinfo=ZoneInfo('UTC'))
+    result = utc_dt.astimezone(None)  # Explicit None
+
+    assert isinstance(result, datetime_tz)
+    # Should convert to local timezone
+    assert result.tzinfo is not None
+    assert result.tzinfo != ZoneInfo('UTC')  # Should not be UTC anymore
+
+    # Test with datetime_cet
+    cet_dt = datetime_cet(2021, 1, 1, 10, 0, 0, tzinfo=ZoneInfo('CET'))
+    result_cet = cet_dt.astimezone(None)  # Explicit None
+
+    assert isinstance(result_cet, datetime_tz)
+    # Should convert to local timezone
+    assert result_cet.tzinfo is not None
+    assert result_cet.tzinfo != ZoneInfo('CET')  # Should not be CET anymore
+
+    # Test with datetime_tz
+    tz_dt = datetime_tz(2021, 1, 1, 10, 0, 0, tzinfo=ZoneInfo('UTC'))
+    result_tz = tz_dt.astimezone(None)  # Explicit None
+
+    assert isinstance(result_tz, datetime_tz)
+    # Should convert to local timezone
+    assert result_tz.tzinfo is not None
+    assert result_tz.tzinfo != ZoneInfo('UTC')  # Should not be UTC anymore
+
+    # Verify that calling with None is equivalent to calling without arguments
+    utc_dt2 = datetime_utc(2021, 1, 1, 10, 0, 0, tzinfo=ZoneInfo('UTC'))
+    result_no_args = utc_dt2.astimezone()
+    result_explicit_none = utc_dt2.astimezone(None)
+
+    assert result_no_args.tzinfo == result_explicit_none.tzinfo
+    assert result_no_args.timestamp() == result_explicit_none.timestamp()
```

Comment:
These are all copies of the above tests, but here you explicitly pass None instead of assuming the default argument. Maybe you can keep one of them, but add it to the test above?